pkg/controllers/termination/terminate.go

```diff
@@ -102,31 +96,27 @@ func (t *Terminator) terminate(ctx context.Context, node *v1.Node) error {
 
 // getPods returns a list of pods scheduled to a node based on some filters
```

Comment:
Can you change the comment to be more accurate since it's merged the `getEvictablePods()` call into here? 

```suggestion
// getPods returns a list of evictable pods scheduled to a node based on some filters
```